src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,55 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        writeToFile(toFileName, createReport(readFromFile(fromFileName)));
+    }
+
+    private int[] readFromFile(String fromFileName) {
+        try (BufferedReader inputFile = new BufferedReader(new FileReader(fromFileName))) {
+            final String coma_separator = ",";
+            final String supply_word = "supply";
+            final String buy_word = "buy";
+            final int operation_type_index = 0;
+            final int amount_index = 1;
```

Comment:
please move this variables to class field constants, add static modifier. also remember about naming convention for the constants.